app/lib/common/utils/logging.dart

```diff
@@ -8,29 +8,37 @@ Future<void> initLogging() async {
   Logger.root.level = Level.INFO;
   final sdk = await ActerSdk.instance;
   Logger.root.onRecord.listen((record) {
-    debugPrint('${record.level.name}: ${record.time}: ${record.message}');
-    String level = 'trace';
-    if (record.level == Level.WARNING) {
-      level = 'warn';
-    } else if (record.level == Level.SEVERE || record.level == Level.SHOUT) {
-      level = 'error';
-    } else if (record.level == Level.INFO) {
-      level = 'info';
-    } else if (record.level == Level.CONFIG) {
-      level = 'debug';
+    final loggerName = record.loggerName;
+    final level = record.level;
+    final time = record.time;
+    final message = record.message;
+    debugPrint('[$loggerName][${level.name}]: $time: $message');
+
+    String levelName = 'trace';
+    if (level == Level.WARNING) {
+      levelName = 'warn';
+    } else if (level == Level.SEVERE || level == Level.SHOUT) {
+      levelName = 'error';
+    } else if (level == Level.INFO) {
+      levelName = 'info';
+    } else if (level == Level.CONFIG) {
+      levelName = 'debug';
     }
```

Comment:
new switch for the win?!?
```dart
final levelName = switch (level) {
  Level.WARNING => 'warn',
  Level.SEVERE | Level.SHOUT => 'error',
  Level.INFO => 'info',
  Level.CONFIG => 'debug',
};
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2125/commits/fc817e1132c9464f3b83788e6eff66814ca14ecf

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2125/commits/fc817e1132c9464f3b83788e6eff66814ca14ecf